### ISSUE TITLE
Language tokens for VGUI screens (computer terminals) should be added.

### ISSUE THREAD

**Issue by sebastiannielsen** [loiste-interactive/infra-issues#511]
language tokens for these types of screens should be added:
![20160320224430_1](https://cloud.githubusercontent.com/assets/1034930/13907394/6b57adee-eeed-11e5-8344-a6e909755d04.jpg)

To aid in translating the game. I readed a little on valves VGUI documentation (since I know these screens are in-game VGUI panels) and there is a localization support for VGUI panels, so its just up to the VGUI panel creators to actually implement localization support.

**Comment by sebastiannielsen** [comment]
Here is a patch, I did some of the work. This patch also solves issue #512 
[patch.zip](https://github.com/loiste-interactive/infra-issues/files/181592/patch.zip)

**Comment by AbueloNemesis** [comment]
As someone who already translated most of these VGUI screens, I'm concerned if the implementation of this would break all my work.

**Comment by sebastiannielsen** [comment]
@AbueloNemesis  No. Implementation of this means blanking out all text on the current VGUI screens and adding the language tokens as labels.
This action will be back compatible with your translation, as you can just leave the language tokens blank or containing just a space (" ") and use the translated textures in the meantime.

And then, when its released, you can easily port over the translation, which are as easy as copying over the translated text from your source photoshop files to the language tokens inside like INFRA_[lang].txt
Adding this change would make it MUCH easier for translators to translate the game, and bring more translations on the boat, since translating the game is then as simple as editing a text file.

Not everyone cannot afford Photoshop, and other tools required to be able to translate textures, so I think its better if the door could be open for everyone, not just those that own photoshop.

**Comment by AbueloNemesis** [comment]
Ok, thanks for explaining it.
